src/WCSimWCTrigger.cc

```diff
@@ -76,6 +76,9 @@ void WCSimWCTriggerBase::GetVariables()
     exit(-1);
   }
 
+  event_number = 1;
+  output_txt = false;
```

Comment:
Probably worth making this a parameter that's settable in the .mac file. WCSimWCDAQMessenger is the place to do it.